### PR TITLE
Update min-web-api.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/fundamentals/minimal-apis.md](https://github.com/dotnet/AspNetCore.Docs/blob/fdd067f570e30f806635a0e319d7d2a65acd7c98/aspnetcore/fundamentals/minimal-apis.md) | [Minimal APIs quick reference](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis?branch=pr-en-us-30892) |
| [aspnetcore/tutorials/min-web-api.md](https://github.com/dotnet/AspNetCore.Docs/blob/fdd067f570e30f806635a0e319d7d2a65acd7c98/aspnetcore/tutorials/min-web-api.md) | [[Visual Studio](#tab/visual-studio)](https://review.learn.microsoft.com/en-us/aspnet/core/tutorials/min-web-api?branch=pr-en-us-30892) |

### CODE REVIEW

File: aspnetcore/tutorials/min-web-api.md [hunk 1]
```diff
@@ -628,7 +628,7 @@ Verify you can post and get all fields except the secret field.
 
 ### Configure JSON serialization options
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/min-web-api.md:629]: I really don't like all the H3's, I think a bullet list would be better and more maintainable.

File: aspnetcore/tutorials/min-web-api.md [hunk 2]
```diff
@@ -640,16 +640,20 @@ For an example of testing a minimal API app, see [this GitHub sample](https://gi
...
 
 For information on how to deploy to Azure, see [Quickstart: Deploy an ASP.NET web app](/azure/app-service/quickstart-dotnetcore).
 
+### Organize ASP.NET Core Minimal APIs
+
+See [Organizing ASP.NET Core Minimal APIs](https://www.tessferrandez.com/blog/2023/10/31/organizing-minimal-apis.html)
+
 ### Learn more
 
 For more information about minimal API apps, see <xref:fundamentals/minimal-apis>.
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/min-web-api.md:655]:
```suggestion
See <xref:fundamentals/minimal-apis>
```

File: aspnetcore/tutorials/min-web-api.md [hunk 3]
```diff
@@ -640,16 +640,20 @@ For an example of testing a minimal API app, see [this GitHub sample](https://gi
...
-For information on how to use OpenAPI with minimal API apps, see [OpenAPI support in minimal APIs](xref:fundamentals/minimal-apis/openapi).
+See [OpenAPI support in minimal APIs](xref:fundamentals/minimal-apis/openapi).
 
 ### Publish to Azure
 
 For information on how to deploy to Azure, see [Quickstart: Deploy an ASP.NET web app](/azure/app-service/quickstart-dotnetcore).
 
+### Organize ASP.NET Core Minimal APIs
+
+See [Organizing ASP.NET Core Minimal APIs](https://www.tessferrandez.com/blog/2023/10/31/organizing-minimal-apis.html)
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/min-web-api.md:651]: @TessFerrandez

File: aspnetcore/tutorials/min-web-api.md (hunk 1 above)
Comment (tdykstra) [reply aspnetcore/tutorials/min-web-api.md:629]: Bulleted list here makes sense to me.

File: aspnetcore/tutorials/min-web-api.md [hunk 4]
```diff
@@ -640,16 +640,20 @@ For an example of testing a minimal API app, see [this GitHub sample](https://gi
...
+
 ### Learn more
 
-For more information about minimal API apps, see <xref:fundamentals/minimal-apis>.
+See <xref:fundamentals/minimal-apis>
```
Comment (tdykstra) [inline aspnetcore/tutorials/min-web-api.md:655]: You could consider moving this to the first of the links under Next Steps.

**Review by tdykstra — APPROVED** [review]

**Pull request merged**